.travis.yml

```diff
@@ -49,4 +48,12 @@ jobs:
 
     # AGP 3.4.0 E2E tests
     - name: AGP 3.4.0 E2E tests
-      script: bundle exec maze-runner -c --verbose
+      env:
+        - APP_FIXTURE_DIR=features/fixtures/app_agp340
+        - RN_FIXTURE_DIR=features/fixtures/rnapp_agp340/android
+        - NDK_FIXTURE_DIR=features/fixtures/ndkapp_agp340
+      script: bundle exec maze-runner features/agp3-features features/agp-features -c --verbose
```

Comment:
The Ruby Maze Runner tests use tags for version specific tests; I'm not sure it makes sense for this as there's no overlap currently, but it might be worth considering in case that changes in the future

e.g. https://github.com/bugsnag/bugsnag-ruby/blob/master/features/rails_features/user_info.feature#L3
https://github.com/bugsnag/bugsnag-ruby/blob/master/.buildkite/pipeline.yml#L56

Reply:
The current approach is what I've discussed with Alex so I'll stick with this for now - tags might make sense in the future if more versions are added to the test fixtures.